mne/viz/ica.py

```diff
@@ -735,7 +735,7 @@ def plot_ica_overlay(ica, inst, exclude=None, picks=None, start=None,
     start : int
         X-axis start index. If None from the beginning.
     stop : int
```

Comment:
you should also fix the docstring at it can be int | None

same for start above.

Reply:
Thanks!